qutebrowser/browser/downloads.py

```diff
@@ -744,20 +741,78 @@ def fetch(self, reply, fileobj=None, filename=None, auto_remove=False):
 
         return download
 
+    def raise_no_download(self, index):
+        """Raise an exception that the download doesn't exist
+
+        Args:
+            index: The index of the download
+        """
+        if not index:
+            raise cmdexc.CommandError("There's no download!")
+        raise cmdexc.CommandError("There's no download {}!".format(index))
```

Comment:
I suggest renaming `index` to `count` to avoid confusion (count starts with 1, indices start with 0).